### PR TITLE
[zh] translate concepts/Topology Aware Hints

### LINKED ISSUE #27798: [zh] translate concepts/Topology Aware Hints
**This is a Feature Request**

translate /docs/concepts/services-networking/topology-aware-hints.md into Chinese

**What would you like to be added**

/zh/docs/concepts/services-networking/topology-aware-hints.md

### PR DESCRIPTION
**Pull request by zhiguo-lu**
translate /docs/concepts/services-networking/topology-aware-hints.md into Chinese

close #27798

### CODE REVIEW

**Comment by zhiguo-lu** [comment]
https://deploy-preview-27800--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/services-networking/topology-aware-hints/

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+for how clients should consume endpoints. This approach adds metadata to enable
+consumers of EndpointSlice and / or and Endpoints objects, so that traffic to
+those network endpoints can be routed closer to where it originated.
+
+For example, you can route traffic within a locality to reduce
+costs, or to improve network performance.
+-->
+_拓扑感知提示_ 包含客户怎么使用服务端点的建议，从而实现了拓扑感知的路由功能。
+这种方法添加了元数据，以启用 EndpointSlice 和/或 Endpoints 对象的调用者，
+这样，访问这些网络端点的流量就可以在它的起始点附近就近路由。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:31]:
```suggestion
这样，访问这些网络端点的请求流量就可以在它的发起点附近就近路由。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+例如，你可以在一个地点内部路由流量，以降低通信成本，或提高网络性能。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:33]:
```suggestion
例如，你可以在一个地域内路由流量，以降低通信成本，或提高网络性能。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+it originated from. This concept is commonly referred to as "Topology Aware
+Routing". When calculating the endpoints for a {{< glossary_tooltip term_id="Service" >}},
+the EndpointSlice controller considers the topology (region and zone) of each endpoint
+and populates the hints field to allocate it to a zone.
+Cluster components such as the {{< glossary_tooltip term_id="kube-proxy" text="kube-proxy" >}}
+can then consume those hints, and use them to influence how traffic to is routed
+(favoring topologically closer endpoints).
+-->
+Kubernetes 集群越来越多的部署到多区域环境中。
+_拓扑感知提示_ 提供了一种把流量限制在它的起始区域之内的机制。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:54]:
```suggestion
_拓扑感知提示_ 提供了一种把流量限制在它的发起区域之内的机制。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+-->
+## 工作原理 {#implementation}
+
+<!-- 
+The functionality enabling this feature is split into two components: The
+EndpointSlice controller and the kube-proxy. This section provides a high level overview
+of how each component implements this feature.
+-->
+此特性启用的功能分为两个组件：EndpointSlice 控制器和 kube-proxy。
+本节提供每个组件怎么实现此特性的概述。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:90]:
```suggestion
本节概述每个组件如何实现此特性。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+### kube-proxy {#implementation-kube-proxy}
+
+<!-- 
+The kube-proxy component filters the endpoints it routes to based on the hints set by
+the EndpointSlice controller. In most cases, this means that the kube-proxy is able
+to route traffic to endpoints in the same zone. Sometimes the controller allocates endpoints
+from a different zone to ensure more even distribution of endpoints between zones.
+This would result in some traffic being routed to other zones.
+-->
+kube-proxy 组件依据 EndpointSlice 控制器设置的提示，过滤它路由用到的端点。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:153]:
```suggestion
kube-proxy 组件依据 EndpointSlice 控制器设置的提示，过滤由它负责路由的端点。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+  拓扑感知提示功能不可用。
+  你可以在一个集群的不同服务中使用这两个特性，但不能在同一个服务中这么做。
+
+<!-- 
+* This approach will not work well for Services that have a large proportion of
+  traffic originating from a subset of zones. Instead this assumes that incoming
+  traffic will be roughly proportional to the capacity of the Nodes in each
+  zone.
+-->
+* 这种方法不适用于大部分流量来自于区域子集的服务。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:244]:
```suggestion
* 这种方法不适用于大部分流量来自于一部分区域的服务。
```

File: content/zh/docs/concepts/services-networking/topology-aware-hints.md [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+  相反的，这里假设入站流量将根据每个区域中节点的服务能力按比例的分配。
+
+<!-- 
+* The EndpointSlice controller ignores unready nodes as it calculates the
+  proportions of each zone. This could have unintended consequences if a large
+  portion of nodes are unready.
+-->
+* EndpointSlice 控制器在计算每一个区域的容量比例时，会忽略未就绪的节点。
+  这在大量节点未就绪的场景下，会带来非预期的结果。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/topology-aware-hints.md:253]:
```suggestion
  在大量节点未就绪的场景下，这样做会带来非预期的结果。
```

**Pull request merged**